.kokoro/autorelease.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -e
+
+# Make sure secrets are loaded in a well known localtion before running releasetool
+source ./populatesecrets.sh
+populate_all_secrets
+
+declare -r github_token="$SECRETS_LOCATION/google-api-dotnet-client-github-token"
+declare -r nuget_token="$SECRETS_LOCATION/google-apis-nuget-api-key"
+
+now="$(date +%Y-%m-%d.%H-%M-%S)"
+branchname="release-$now"
+cd ..
+
+# Create a new branch to push the changes to.
+git switch -c $branchname
+
+# Download, generate, build and pack all generated libraries
+# Build support libraries in case the latest support library version isn't yet on nuget.
+
+./BuildSupport.sh
+
+./BuildGenerated.sh --onlydownload
+
+# Delete broken discovery docs here.
+# E.g. rm -f DiscoveryJson/streetviewpublish_v1.json
+
+# These have nested resources with the same type as the parent
+rm -f DiscoveryJson/sasportal_v1alpha1.json
+rm -f DiscoveryJson/prod_tt_sasportal_v1alpha1.json
+rm -f DiscoveryJson/apigee_v1.json
+# The following 3 have a Service named parameter in a request
+# that clashes with our own IClientService Service property
+# that we have in BaseServiceRequest
+rm -f DiscoveryJson/metastore_v1alpha.json
+rm -f DiscoveryJson/metastore_v1beta.json
+rm -f DiscoveryJson/metastore_v1.json
+# b/238627322
+rm -f DiscoveryJson/monitoring_v1.json
+# We've only generated identitytoolkit_v3 in 2014.
```

Comment:
I don't understand this comment... it sounds like it would be fine to keep generating and publishing v3. Is there a reason we're still deleting v3?

Reply:
Oh no, it's that we only generated it up to 2014, it was skipped for generation after that and the package delisted: https://www.nuget.org/packages/Google.Apis.IdentityToolkit.v3/.

I've asked the Auth team about whether we should generate v2 and v3 and they are looking into it.

I'm changing this to "during" to make it clearer.